src/platform/site-wide/user-nav/selectors.js

```diff
@@ -13,6 +14,16 @@ export const selectUserGreeting = createSelector(
       return startCase(toLower(name.first || sessionFirstName));
     }
 
```

Comment:
@ncksllvn Made the changes you suggested. Heer is my solution for it. I had to use a span because it worked correctly with the foundation responsive utility classes, without having to fiddle with the css too much.